docs/build/guides/conventions/stellar-asset-contract.mdx

```diff
@@ -0,0 +1,94 @@
+    // Call token operations part of the SEP-41 token interface
+    client.transfer(...);
+  }
+}
+```
+
+The address parameter is not the address of the issuer of an asset, it corresponds to the deployed contract address for this asset.
+
+```bash
+stellar contract id asset \
+    --source G... \
+    --network testnet \
+    --asset [asset:issuer]
+```
+
+E.g. for USDC, it would be `--asset USDC:G...` For the native asset, XLM, `--asset native`. See the [deploy SAC] guide for more details.
+
+[deploy SAC]: ../cli/deploy-stellar-asset-contract.mdx
+
+:::info Clients
+
+A client created by [`token::Client`] implements the functions defined by any contract that implements the [SEP-41 Token Interface]. But with [CAP-46-6 smart contract standardized asset], the Stellar Asset Contract exposes additional functions such as `mint`. To access the additional functions, another client needs to be used: [`token::StellarAssetClient`]. This client only implements the functions from CAP-46-6, which are not part of the SEP-41.
+
+```rust
+let client = token::StellarAssetClient::new(&env, &id);
+// Call token operations which are not part of the SEP-41 token interface
+// but part of the CAP-46-6 Smart Contract Standardized Asset
+client.mint(...);
+```
+
+:::
+
+### Testing
+
+Soroban Rust SDK provides an easy way to instantiate a Stellar Asset Contract tokens using `register_stellar_asset_contract`. This function can be seen as the deployment of a generic token. In the following example, we are following the best practices outlined in the [Issuing and Distribution Accounts section](../../../tokens/control-asset-access.mdx#issuing-and-distribution-accounts):
+
+```rust
+#![cfg(test)]
+
+use soroban_sdk::testutils::Address as _;
+use soroban_sdk::{token, Address, Env};
+use token::TokenClient;
+use token::StellarAssetClient;
+
+#[test]
+fn test() {
+    let e = Env::default();
+    e.mock_all_auths();
+
+    let issuer = Address::random();
+    let distributer = Address::random();
+
+    let token_address = e.register_stellar_asset_contract(issuer.clone());
+
+    // client for SEP-41 operations
+    let token = TokenClient::new(&e, &token_address);
+    // client for Stellar Asset Contract operations
+    let token_sac = StellarAssetClient::new(&e, &token_address);
+
+    // 1e7 diff between minimal unit and unit itself
+    let genesis_amount: i128 = 1_000_000_000 * 10_000_000;
+
+    token_sac.mint(&issuer, &distributer, &genesis_amount);
+
+    assert_eq!(token.balance(&distributor), genesis_amount);
+```
+
+### Examples
+
+See the full examples that utilize the token contract in various ways for more details:
+
+- [Timelock](../../smart-contracts/example-contracts/timelock.mdx) and [single offer](../../smart-contracts/example-contracts/single-offer-sale.mdx) move token via `xfer` to and from the contract
+- [Atomic swap](../../smart-contracts/example-contracts/atomic-swap.mdx) uses `incr_allow` to transfer token on behalf of the user
```

Comment:
This example no longer uses `allowance` functions, but does use the `transfer` function. The text on that page looks like it's pretty out-dated and will need to be updated sometime soon.

I quickly looked through the example contracts, and it looks like there isn't really any use of the `approve` or `allowance` functions, outside of the `token` example and a test or two.

Reply:
For now I said it's `transfer` too and changed the references to `incr_allow` as not used in the example.